apps/web/src/lib/studies.ts

```diff
@@ -266,10 +266,30 @@ export const getStudiesForExport = async (organisationIds: number[]) => {
 
 export type UpdateStudyInput = Prisma.StudyUpdateInput
 
+export const mapCPMSStatusToSEStatus = (cpmsStatus: string): string => {
+  const statusMap: { [key: string]: string } = {
+    'Pre-Setup': 'In etup',
```

Comment:
Minor, spelling mistake - assuming this should be `'In setup'` ?

Reply:
updated